### PR TITLE
Debug pages changes: separate endpoints, new page with sync

### PR DESCRIPTION
**Pull request by mm-near**
* added new page with more detailed sync status
* added information about shard sizes
* refactored the code, so that we have separate endpoints to fetch different information

### CODE REVIEW

**Review by nikurt — APPROVED** [review]
I'm not the best person for an in-depth review of the javascript part of the PR, but the rest of the PR is straight-forward and looks good.

File: chain/client/src/client_actor.rs [hunk 1]
```diff
@@ -710,8 +764,172 @@ impl ClientActor {
...
+    ) -> Result<Vec<EpochInfoView>, near_chain_primitives::Error> {
+        // Next epoch id
+        let mut epochs_info: Vec<EpochInfoView> = Vec::new();
+
+        if let Ok(next_epoch) = self.get_next_epoch_view() {
+            epochs_info.push(next_epoch);
+        }
+        let head = self.client.chain.head()?;
+        let mut current_block = head.last_block_hash;
+        for _ in 0..5 {
```
[the remark this answers is not in the document]
Comment (mm-near) [reply chain/client/src/client_actor.rs:812]: done

File: chain/client/src/client_actor.rs [hunk 2]
```diff
@@ -710,8 +764,172 @@ impl ClientActor {
...
+
+        let mut blocks_debug: Vec<DebugBlockStatus> = Vec::new();
+        let mut last_block_hash = head.last_block_hash;
+        let mut last_block_timestamp: u64 = 0;
+        let mut last_block_height = head.height + 1;
+
+        let initial_gas_price = self.client.chain.genesis_block().header().gas_price();
+
+        // Fetch last 50 blocks (we can fetch more blocks in the future if needed)
+        for _ in 0..50 {
```
[the remark this answers is not in the document]
Comment (mm-near) [reply chain/client/src/client_actor.rs:837]: done

File: chain/jsonrpc/res/epoch_info.html [hunk 3]
```diff
@@ -49,65 +49,133 @@
...
                     });
+                    $('.js-tbody-shard-sizes').append(row);
+                }
+                maxShards = Math.max(epoch.shards_size_and_parts.length, maxShards);
+            });
+
+
+            validatorMap.forEach((value, key) => {
+                let row = $('<tr>').append($('<td>').append(key));
+                for (i = 0; i < epoch_data.status_response.EpochInfo.length; i = i + 1) {
```
[the remark this answers is not in the document]
Comment (mm-near) [reply chain/jsonrpc/res/epoch_info.html:130]: done

File: chain/jsonrpc/res/epoch_info.html [hunk 4]
```diff
@@ -143,6 +211,22 @@ <h2> Validators </h2>
...
+
+            </tr>
+        </thead>
+        <tbody class="js-tbody-shard-sizes">
+        </tbody>
+    </table>
+    Orange color means, that a given shard was requested by other peer for syncing.
 </body>
 
 </html>
```
[the remark this answers is not in the document]
Comment (mm-near) [reply chain/jsonrpc/res/epoch_info.html:232]: seems that my editor keeps removing it .. not sure if it is needed for HTML

**Pull request merged**